scripts/all.sh

```
@@ -54,23 +53,23 @@ for buildType in "Debug" "Release"; do
     if [ $buildType == "Release" ]; then
 
         echo -e "${GREEN}Running build and test stock queries${NORMAL_OUTPUT}"
-        ./scripts/build_and_test_stock_queries.sh -b ${buildType} $@
+        ./scripts/queries/base_queries/build_and_test_stock_queries.sh -b ${buildType} $@
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)